jobs/ci-kubernetes-e2e-gce-large-correctness.env

```diff
@@ -0,0 +1,28 @@
+### provider-env
+KUBERNETES_PROVIDER=gce
+PROJECT=kubernetes-scale
+KUBE_GCE_ZONE=us-east1-a
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+### cluster-env
+NUM_NODES=2000
+MASTER_SIZE=n1-standard-32
```

Comment:
I believe we should be able to drop that - it's automatically computes from number of Nodes.